spec/rails_helper.rb

```diff
@@ -1,23 +1,18 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
-ENV['RAILS_ENV'] ||= 'test'
-require File.expand_path('../config/environment', __dir__)
+ENV["RAILS_ENV"] ||= "test"
+require File.expand_path("../config/environment", __dir__)
 
-require 'spec_helper'
-require 'capybara/rails'
+require "spec_helper"
+require "capybara/rails"
 require "capybara/rspec"
 require "selenium/webdriver"
 require "webdrivers"
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 Dir[Rails.root.join("lib/strategies/**/*.rb")].each { |f| require f }
 
-
-Capybara.javascript_driver = :selenium_chrome_headless
-Capybara.server = :puma, { Silent: true }
-Capybara.default_max_wait_time = 0.2
```

Comment:
I suspect we will need something like these again when we want to try `js: true` again... but until then this makes sense to remove. ✅